Pieces.py

```diff
@@ -0,0 +1,33 @@
+class Piece:
+    location = (column, row)
+    def __init__(self, team):
+        self.team = team
+        self.location = location
+        self.type = type
+
+    def get_location(self, location):
+        return location
+class King:
```

Comment:
Will `King` inherit from `Piece`?

Reply:
Yes, but I am starting with everything just as a class. Pawn, Castle, Rook, Bishop, King, and Queen classes will inherit from Pieces once I have some more structure